.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
```

Comment:
Looks like we are duplicating work: https://github.com/cdklabs/aws-delivlib/pull/1551

You should be upgrading setup-node and checkout versions as well.

Reply:
Sorry. Do you want to take over? On your PR, it looks like you manually updated the files whereas you'll need to update the `projenrc.ts` file for any changes to stick.

Reply:
These versions are controlled by projen. I don't think they are necessary to fix the broken workflow atm

Reply:
There isn't much to take over here...

A little confused why projen is needed here but it's how it is setup.

The error (which I can't find anymore) said checkout and node actions needed upgrading to node v20. Which I don't think CDK even supports.